docs/releasing.md

```diff
@@ -4,11 +4,13 @@ This document describes the steps to release a new version of the Dynatrace dist
 
 ## Collector release prerequisites
 
-Ensure the manifest contains all desired collector components.
+Before starting a release, make a PR against to verify and update the following:
 
-Collector components are controlled by the same `manifest.yaml`.
-It is recommended that the component versions depend on the same version of the collector.
-For components in the upstream collector repos, most of the time this means they will be the same version as the collector version.
+1. Ensure `manifest.yaml` contains all desired Collector components at the desired version. It is recommended that the component versions depend on the same minor version of the Collector. For components in the upstream Collector repos, most of the time this means they will be the same version as the Collector version. It is okay for bugfix versions to differ.
```

Comment:
Nit: Maybe break the line so it's easier to add comments in PR reviews

Reply:
Good call. Done.